### ISSUE TITLE
How to generate training/validation list.

### ISSUE THREAD

**Issue by meanmee** [yl-1993/hfsoftmax#2]
Hi， It's a great work， Could you tell me how to generate training/validation list since the rec2ims.py didn't generate these list files.

**Comment by yl-1993** [comment]
@meanmee You can randomly split 5% or 10% as validation set and keep the rest data as the training set, since there is no strict protocol to split the dataset.
